draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -299,7 +299,7 @@ The identifying index of a leaf node is relative to all nodes in the tree size f
 
 ### Receipt of Inclusion
 
-In a signed inclusion proof, the previous merkle tree root, maps to tree-size-1, and is a detached payload.
+In a signed inclusion proof, the payload is the merkle tree root which corresponds to the log at size tree-size.
```

Comment:
seeing size tree-size was a bit confusing. Suggestion to change code fragments use `code` formatting.
```suggestion
In a signed inclusion proof, the payload is the merkle tree root which corresponds to the log at size `tree-size`.
```